library/billing/src/HCFA_Info.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * Helper class to manage rows and columns for the information on
+ * the HCFA 1500 02/12 claim form.
+ * This allows "out of order" creation of the content.
+ *
+ * @package OpenEMR
+ * @author Kevin Yeh <kevin.y@integralemr.com>
+ * @author Stephen Waite <stephen.waite@cmsvt.com>
+ * @copyright Copyright (c) 2013 Kevin Yeh <kevin.y@integralemr.com>
+ * @copyright Copyright (c) 2018 Stephen Waite <stephen.waite@cmsvt.com>
+ * @link https://github.com/openemr/openemr/tree/master
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace OpenEMR\Billing;
+
+use OpenEMR\Billing\HCFA_1500;
+
+class HCFA_Info
+{
+    protected $row;
+    protected $column;
+    protected $width;
+    protected $info;
```

Comment:
When things are working, then can change above variables from protected to private

Reply:
Actually, did some reading on this. Using protected is just fine (just in case anybody ever extends this class).